python/dgl/graphbolt/dataset.py

```diff
@@ -117,18 +122,21 @@ class OnDiskDataset(Dataset):
     def __init__(self, path: str) -> None:
         with open(path, "r") as f:
             self._meta = OnDiskMetaData.parse_raw(f.read(), proto="yaml")
+        self._train_set = self._init_tvt_sets(self._meta.train_set)
```

Comment:
If the train_set can be a list of OnDiskTVTSet, should we use plural for train_set(s) and others?

Reply:
sure